HopsanGUI/HcomHandler.cpp

```diff
@@ -8881,7 +8881,9 @@ void HcomHandler::executeLtBuiltInFunction(QString fnc_call)
             pVar2 = mAnsVector;
         }
 
-        LogDataHandler2 *pLogDataHandler = mpModel->getViewContainerObject()->getLogDataHandler().data();
+        LogDataHandler2 *pLogDataHandler = 0;
+        if(mpModel)
+            pLogDataHandler = mpModel->getViewContainerObject()->getLogDataHandler().data();
```

Comment:
pLogDataHandler is used on line 8907 and 8917, so if it is nullptr it will still crash
But maybe those cant happen if no model i opened, (pVar1,2) will be nullptr

Reply:
I cannot happen, because evaluating arg1 and arg2 will not result in data vector answers unless there is an open model. But I will add checks anyway to get some redundancy.